### PR TITLE
Misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -i 3 -w --skip="./thirdparty,./share/locale,./fonts" -I ../musescore-whitelist.txt`  
Whitelist:
```
clas
nd
ot
pres
strack
tage
te
```

### CODE REVIEW

File: mscore/pluginManager.ui [hunk 1]
```diff
@@ -126,7 +126,7 @@
          <widget class="QPushButton" name="reloadPlugins">
           <property name="toolTip">
            <string>Reload all plugins.
-This will re-evaluate all plugins, picking up any changes that may have occured.</string>
+This will re-evaluate all plugins, picking up any changes that may have occurred.</string>
```
Comment (Jojo-Schmitz) [inline mscore/pluginManager.ui]: At least this one needs to go to 2.2 too

**Comment by Jojo-Schmitz** [comment]
Seems you need to sign the [CLA](https://musescore.org/cla)

**Comment by luzpaz** [comment]
Signed under 'Kunda1'

**Comment by lasconic** [comment]
Thank you and welcome in the MuseScore contributor community! Looking forward to your next contribution !

**Pull request merged**